content/doc/developer/publishing/preparation.adoc

```diff
@@ -32,8 +32,10 @@ See link:../documentation[Documentation] for more information.
 
 == Signup Required
 
-Jenkins plugins distributed via Jenkins project update sites need to be hosted in the https://github.com/jenkinsci[jenkinsci GitHub organization], so you will need a user account on GitHub.
+Jenkins plugins distributed via Jenkins project update sites need to be hosted in the https://github.com/jenkinsci[jenkinsci GitHub organization], so you will need a user account on GitHub. 
 
-To actually release your plugin, you will need a https://accounts.jenkins.io[Jenkins community account] that will give you access to the https://issues.jenkins.io/[Jira issue tracker] and https://repo.jenkins-ci.org/[Maven repository].
+To actually release your plugin, you will need a https://accounts.jenkins.io[Jenkins community account] that will give you access to the https://issues.jenkins.io/[Jira issue tracker] and https://repo.jenkins-ci.org/[Maven repository]. 
+
+You will receive a welcome email if your Jenkins community account was created successfully. If you do not receive an email, open a ticket at https://github.com/jenkins-infra/helpdesk.
```

Comment:
```suggestion
You will receive a welcome email if your Jenkins community account was created successfully.
If you do not receive an email, open a ticket at link:https://github.com/jenkins-infra/helpdesk[the Jenkins helpdesk].
```
small item to make sure each sentence has its own line, making sure the `link:` macro is used